CaloMC/fcl/common.fcl

```diff
@@ -1,6 +1,6 @@
 BEGIN_PROLOG
-  HitMakerDigitizationStart    : 450.
-  HitMakerDigitizationEnd      : 2005.0
+  HitMakerDigitizationStart    : 250.
```

Comment:
This seems to include the event window marker offset, which should come from DaqConditions I believe.

Reply:
The way we've been encoding it for the other systems is that the fcl parameterization for the digitization is given in terms of nominal time from protons, and everywhere it is actually used it subtracts out timeFromProtonsToDRMarker from DAQConditions. We can eventually move all of this to DAQConditions and provide a method for getting these times, but for this PR I think it should be done here.